### DISCUSSION TITLE
Should we release 0.71.2?

### DISCUSSION THREAD

**Discussion by Titozzz** [reactwg/react-native-releases#53] [Patches]
## Released, please refer to https://github.com/reactwg/react-native-releases/discussions/57

---

# Should we release 0.71.2?

Current Release: [0.71.1](https://github.com/facebook/react-native/releases/tag/v0.71.1)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/32f54877ff788240d24528d208c704ee78e4e761
2. https://github.com/facebook/react-native/commit/1fef37681298c828a07febcd0d975a32f6bc4403
3. https://github.com/facebook/react-native/commit/71c4f57baf6683ea4304e15c040d6b6c3b3d2b73
4. https://github.com/facebook/react-native/commit/e162b07982cf9481038de71f5dd7bd9b45387f0a
5. https://github.com/facebook/react-native/commit/c28944284894a3188b97e3d8bb5b489755852160
1. https://github.com/facebook/react-native/commit/a8166bd75b221f967a859d5cc25b3394c4d35301
2. https://github.com/facebook/react-native/commit/be69c8b5a77ae60cced1b2af64e48b90d9955be5
3. https://github.com/facebook/react-native/commit/305ca337c0471c61cb74216bd93ae3f1a232a89f
4. https://github.com/facebook/react-native/commit/fac7859863c7130740aacc95d0e62417bd8f789e
5. ~https://github.com/facebook/react-native/commit/a00cea46bf0bde788e6f92ae8f750bd9e32a2739~ (we'll leave it for next patch, it needs some other commits)
1. https://github.com/facebook/react-native/pull/35946
1. https://github.com/facebook/react-native/pull/36004
1. https://github.com/facebook/react-native/pull/36003
2. https://github.com/facebook/react-native/pull/35992

#### Local commits to backport to main

1.

**Comment by NickGerleman** [comment]
1. Fix 0.71 causing invariant violations when nesting VirtualizedList inside ListEmptyComponent inside VirtualizedList https://github.com/facebook/react-native/commit/1fef37681298c828a07febcd0d975a32f6bc4403
2. Add missing type https://github.com/facebook/react-native/commit/71c4f57baf6683ea4304e15c040d6b6c3b3d2b73
3. Add missing type https://github.com/facebook/react-native/commit/e162b07982cf9481038de71f5dd7bd9b45387f0a
4. Make TS happy when directly importing TurboModuleRegistry https://github.com/facebook/react-native/commit/c28944284894a3188b97e3d8bb5b489755852160

**Comment by mmsmadi** [comment]
So far I am facing issues with keyboard listener and images are not shown on android apk while they are working in development.

Also not forget to mention I had to turn of hermes to make development run faster.

**Reply by cortinico** [reply]
> Also not forget to mention I had to turn of hermes to make development run faster.

This was fixed in 0.71.1
Also this thread is only for pick requests of commits that are fixing issues discovered on a release

**Comment by bigcupcoffee** [comment]


**Reply by kelset** [reply]
it's literally listed in the main list: 
<img width="601" alt="Screenshot 2023-01-24 at 10 32 11" src="https://user-images.githubusercontent.com/16104054/214269357-d2bbe501-4e16-4d74-94a6-ea50a143b04d.png">

**Comment by NickGerleman** [comment]
https://github.com/facebook/react-native/pull/35859 also fixes a 0.71 regression, need to push through internally but should be soon, if we want to batch it into this release.

**Reply by NickGerleman** [reply]
https://github.com/facebook/react-native/commit/a8166bd75b221f967a859d5cc25b3394c4d35301

**Comment by NickGerleman** [comment]
Already a side-channel for it, but a reminder that we may be wanting to backport https://github.com/facebook/react-native/pull/35967 (Samsung TextInput hang partial mitigation) soon to more than just 0.71. Depending on timing it may make sense to coalesce into 0.71.2 (or not if it's going to delay things). Right now its targeting EOWish.

**Reply by NickGerleman** [reply]
https://github.com/facebook/react-native/commit/be69c8b5a77ae60cced1b2af64e48b90d9955be5 for reference

**Reply by kelset** [reply]
I've added the reference also to the 0.70, 0.69, 0.68 discussion (pending if we'll do 0.68 or not) - 0.71.2 was already planned for mid-next-week, I think it's safe to assume we'll do all of them around then

**Comment by NickGerleman** [comment]
Another typing fix landed this morning. https://github.com/facebook/react-native/commit/305ca337c0471c61cb74216bd93ae3f1a232a89f

**Comment by cortinico** [comment]
I'd like to get this one included:
- https://github.com/facebook/react-native/pull/35992
(It requires a RNGP bump).

It will solve a build failure caused when importing more than 1 library which is not specifyign `jsRootDir` in New Architecture enabled projects.

**Comment by NickGerleman** [comment]
Another TS addition https://github.com/facebook/react-native/commit/fac7859863c7130740aacc95d0e62417bd8f789e

**Comment by kelset** [comment]
we want this https://github.com/facebook/react-native/commit/a00cea46bf0bde788e6f92ae8f750bd9e32a2739 for this https://github.com/facebook/react-native/issues/35054

**Reply by kelset** [reply]
(but we'll do it next patch release)

**Comment by idrissakhi** [comment]
Hello 
i have a major issue (https://github.com/facebook/react-native/issues/36019) since 0.71 IDK if https://github.com/facebook/react-native/commit/32f54877ff788240d24528d208c704ee78e4e761 will help resolve the issue. But it's really blocking issue.
i had the issue on Redmi 9 Android 10, other versions seems to work just fine
